app/src/main/java/net/kwatts/powtools/model/OWDevice.java

```diff
@@ -9,6 +9,7 @@
 import android.databinding.ObservableInt;
```

Comment:
Nice! Approve and really dig seeing the code line count go down. These type of changes will make it easier to implement another model/device if we need to (the initial idea but got a bit out of hand...)